test/db/tools/rz_bin

```diff
@@ -176,15 +176,15 @@ NAME=rz-bin -zz pe
 FILE=bins/pe/ioli/w32/crackme0x00.exe
 CMDS=!rz-bin -zz ${RZ_FILE} | grep "Password:"
 EXPECT=<<EOF
-81  0x00002619 0x00404019 10  11   .rdata  ascii   Password: 
+102 0x00002619 0x00404019 10  11   .rdata  ascii   Password: 
```

Comment:
And here

Reply:
[crackme.diff.txt](https://github.com/rizinorg/rizin/files/7528416/crackme.diff.txt)


```
+14  0x00000803 0x00401403 8   9    .text   ibm037  eKÍZICý©
+16  0x000008d0 0x004014d0 7   8    .text   ibm037  Ii}CI[\b
+20  0x000009b0 0x004015b0 16  17   .text   ibm037  8hCiOi'ÜiõÈàiQc\
+29  0x00000f19 0x00401b19 6   7    .text   ibm037  ÔbiCZÎ
+30  0x00001020 0x00401c20 6   7    .text   ibm037  IiHC»¢
+34  0x000010b8 0x00401cb8 4   5    .text   ibm037  KYS8
+37  0x000011a7 0x00401da7 7   8    .text   ibm037  iBiQY07
+38  0x000011dc 0x00401ddc 7   8    .text   ibm037  iBiQY]7
+41  0x000012a8 0x00401ea8 7   8    .text   ibm037  i8JYiCÔ
+44  0x00001371 0x00401f71 7   8    .text   ibm037  iá\iHAY
+46  0x000013a4 0x00401fa4 5   6    .text   ibm037  iQY55
+49  0x000015d6 0x004021d6 5   6    .text   ibm037  iQYC3
+50  0x000015ed 0x004021ed 6   7    .text   ibm037  KiHY¿3
+51  0x000015f5 0x004021f5 5   6    .text   ibm037  iQYm2
+54  0x000016e4 0x004022e4 7   8    .text   ibm037  ¶QiQY38
+55  0x000016fa 0x004022fa 7   8    .text   ibm037  iBiQY¸2
+56  0x00001711 0x00402311 6   7    .text   ibm037  KiQYg2
+67  0x000019ab 0x004025ab 7   8    .text   ibm037  i)qiQc\
+73  0x00001b97 0x00402797 5   6    .text   ibm037  iHYb8
+74  0x00001c98 0x00402898 7   8    .text   ibm037  ¶Qi2iQY
+75  0x00001cb3 0x004028b3 7   8    .text   ibm037  iBiQYUÖ
```